src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticLessAndGreaterThanCompletionTests.cs

```diff
@@ -135,23 +135,6 @@ public void Multiple_Nested()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.AutomaticCompletion)]
-        public void TypeArgument_Invalid()
-        {
-            var code = @"class C
-{
-    void Method()
-    {
-        List$$
-    }
-}";
-            using (var session = CreateSession(code))
-            {
-                Assert.NotNull(session);
-                CheckStart(session.Session, expectValidSession: false);
-            }
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.AutomaticCompletion)]
         public void TypeArgument_Invalid2()
```

Comment:
Just TypeArgument_Invalid now? The TypeArgument# tests below start on 2 as well. Weird.